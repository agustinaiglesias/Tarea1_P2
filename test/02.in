definirFecha 10/02/2023
fechaActualExtendida
Fin