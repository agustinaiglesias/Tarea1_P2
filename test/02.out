1>Fecha definida en forma exitosa.
2>La fecha actual es 10 de Febrero de 2023.
3>Fin.
